src/app/home/pages/PlansPage/components/PlansTable.tsx

```diff
@@ -73,15 +99,24 @@ const PlansTable: React.FunctionComponent<IPlansTableProps> = ({
   useEffect(() => setPageNumber(1), [sortBy]);
 
   const rows = flatten(
-    currentPageItems.map((plan, planIndex) => {
+    currentPageItems.map((plan: IPlan, planIndex) => {
       const {
         planName,
         migrationCount,
         sourceClusterName,
         targetClusterName,
         storageName,
-        pvCount,
+        namespaceCount,
+        isMaxResourcesLimitReached,
       } = getPlanInfo(plan);
+      const isLoadingAnalytic: boolean =
+        !!(plan?.PlanStatus?.analyticPercentComplete !== 100 && plan.PlanStatus.latestAnalytic) ||
```

Comment:
I feel like we need to unwrap this logic to be more explicit, this is pretty gnarly to try to reason through just reading it. Like, just trying to figure out what `isLoadingAnalytic` from this whole boolean expression is not clear.

Reply:
You're right - I will refactor this.